src/docfx/models/schemas/LandingData.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    internal class LandingData
```

Comment:
all fields will output to content? how should we know which filed should be markup?

Reply:
no markdown in LandingData

---
In reply to: [203634660](https://github.com/dotnet/docfx/pull/3098#discussion_r203634660) [](ancestors = 203634660)

Reply:
so bydefault all field will be serialized to output "content"?

---
In reply to: [203635756](https://github.com/dotnet/docfx/pull/3098#discussion_r203635756) [](ancestors = 203635756,203634660)

Reply:
yes

---
In reply to: [203636251](https://github.com/dotnet/docfx/pull/3098#discussion_r203636251) [](ancestors = 203636251,203635756,203634660)